src/field/launch_application_data1.c

```diff
@@ -0,0 +1,19 @@
+#include "global.h"
```

Comment:
Rather than just splitting the data, you should consider splitting the C code based on inferred file boundaries. My original splitting pass is not gospel.

Reply:
Split the main file `field/launch_application.c` according to data. Will address the non-matched `OVY_MGR_TEMPLATE`'s later depending on time and motivation